core/templates/dev/head/pages/topics_and_skills_dashboard/skills_list_directive.html

```diff
@@ -15,28 +15,35 @@
       </th>
     </tr>
     <tr ng-repeat="skill in getSkillSummaries()"
-        class="list-item">
+        ng-click="isInModal() && selectSkill(skill)"
+        class="list-item"
+        ng-init="skillLink=isInModal()? '#':getSkillEditorUrl(skill.id)">
       <td ng-class="{'hoverable-cell': highlightedIndex === $index}"
           ng-mouseover="highlightColumns($index)"
           ng-mouseleave="unhighlightColumns()">
-        <a ng-href="<[getSkillEditorUrl(skill.id)]>" class="list-summary">
+        <a ng-href="{{skillLink}}" class="list-summary">
```

Comment:
Here and elsewhere, use <[...]> not {{...}}. We reserve the latter for Jinja stuff.

Reply:
Done